app/src/main/java/org/simple/clinic/patient/OngoingNewPatientEntry.kt

```diff
@@ -135,11 +135,7 @@ data class OngoingNewPatientEntry(
         errors += dobValidationCheck(dobValidator, dateOfBirth, ageValidator)
```

Comment:
We need to be more detailed in the commit messages. Instead of something like "Refactor `OngoingNewPatientEntry`", we should be saying exactly what is being done. In this case, it will be something like, "Extract validation logic into discrete methods in `OngoingNewPatientEntry`".